app/src/openxr/cpp/OpenXRLayers.h

```diff
@@ -183,12 +183,7 @@ class OpenXRLayerBase : public OpenXRLayer {
     XrSwapchainCreateInfo info{XR_TYPE_SWAPCHAIN_CREATE_INFO};
     info.width = width;
     info.height = height;
-    bool shouldZeroInitialize = aSurfaceType == VRLayerSurface::SurfaceType::AndroidSurface;
-#if defined(PICOXR)
-    // Circumvent a bug in the pico OpenXR runtime.
-    shouldZeroInitialize = false;
-#endif
-    if (shouldZeroInitialize) {
+    if (aSurfaceType == VRLayerSurface::SurfaceType::AndroidSurface) {
```

Comment:
In the short term we should keep the old behaviour for <5.4. I was told that the way to detect the system version is by
```
char buffer[128] = {0};
__system_property_get("[ro.build.id](http://ro.build.id/)", buffer);
__system_property_get("[ro.system.build.id](http://ro.system.build.id/)", buffer);
```